easybuild/framework/easyblock.py

```diff
@@ -188,15 +188,28 @@ def __init__(self, ec):
             self.log.warning("Group spec '%s' is overriding config group '%s'." % (self.cfg['group'], group_name))
             group_name = self.cfg['group']
 
+        # group
         self.group = None
         if group_name is not None:
             self.group = use_group(group_name)
 
+        # module features
+        self.module_family = build_option('module_family')
+        self.module_properties = None
+        if build_option('module_properties') is not None:
+            self.module_properties = []
+            for prop_spec in build_option('module_properties'):
+                prop_spec_parts = prop_spec.split(':')
```

Comment:
if you pass invalid properties, Lmod will complain and the `module load` done during the sanity check will not work, so you'll know